src/components/pages/TodoPage/TodoPage.tsx

```diff
@@ -12,10 +12,14 @@ const TodoPage = () => {
   const { isLoading, todos } = useSelector(todoSelector);
 
   useEffect(() => {
-    dispatch(fetchTodos());
+    if (!todos) {
+      dispatch(fetchTodos());
+    }
   }, [dispatch]);
 
-  return <StyledContainer>{isLoading ? <Loader /> : <Todos todos={todos} />}</StyledContainer>;
+  return (
+    <StyledContainer>{isLoading ? <Loader /> : <Todos todos={todos || []} />}</StyledContainer>
```

Comment:
you can define default value as an array for todos inside <Todos /> component
`..., todos: array = [], ...`